users/views.py

```diff
@@ -101,15 +99,15 @@ def get_success_url(self) -> str:
 class UserUpdate(UpdateView):
     template_name = "user_update.html"
     form_class = SimpleUserChangeForm
-    admins_form_class = AdminUserChangeForm
+    context_object_name = "user_detail"
     model = CustomUser
 
     def get_object(self, queryset: Optional[QuerySet] = None) -> CustomUser:
         return self.request.user
 
-    def get_form_class(self) -> Type[ModelForm]:
-        if self.request.user.user_type == CustomUser.UserType.ADMIN.name:
-            return self.admins_form_class
+    def get_form_class(self) -> Any:
```

Comment:
Typing for this place should be `Union[SimpleUserChangeForm, AdminUserChangeForm]`

Reply:
Yeah, totally unnecessary change on my part.
I wonder though, wouldn't it be better to keep a single, parent type (_ModelForm_) like before rather than a _Union_?